skimage/filters/rank/generic.py

```diff
@@ -457,15 +485,18 @@ def subtract_mean(image, selem, out=None, mask=None, shift_x=False,
                                    shift_x=shift_x, shift_y=shift_y)
 
 
-def median(image, selem, out=None, mask=None, shift_x=False, shift_y=False):
+@default_selem
+def median(image, selem=None, out=None, mask=None,
+           shift_x=False, shift_y=False):
     """Return local median of an image.
 
     Parameters
     ----------
     image : 2-D array (uint8, uint16)
         Input image.
-    selem : 2-D array
-        The neighborhood expressed as a 2-D array of 1's and 0's.
+    selem : 2-D array, optional
+        The neighborhood expressed as a 2-D array of 1's and 0's. If None, a
+        square (or cube in 3-D) of size 3 is used.
```

Comment:
The thing about 3-D is slightly confusing here as the function doesn't declare to support 3-D images.